doc/en/components/pie-chart.md

```diff
@@ -6,11 +6,50 @@ mentionedTypes: ['XamPieChart','SliceSelectionMode','PieSliceOthersContext']
 ---
 # $PlatformShort$ Pie Chart Overview
 
-The $ProductName$ pie chart component is a specialized component that renders a pie chart, consisting of a circular area divided into sections. Each section has arc length proportional to its underlying data value.
+A Pie Chart, or Pie Graph, is a part-to-whole chart that shows how categories (parts) of a data set add up to a total (whole) value. Categories are rendered as sections in a circular, or pie-shaped graph. Each section, or pie slice, has an arc length proportional to its underlying data value. Categories are shown in proportion to other categories based on their value percentage to the total value being analyzed, as parts of 100 or 100%.
```

Comment:
Jason wants to see a sample in each topic after short introduction (1-3 sentences) so there should be a link to a sample here.

Reply:
These is the content provided by Jason, but I absolutely agree with you that we should show a sample above the fold of the page when it loads

Reply:
Here's the starting example

https://share.goabstract.com/9e5d9807-a47c-47c5-ae31-563f4f346f3d?collectionLayerId=2cae6e71-217a-45b4-9b4d-653056a7d078&mode=build